kornia/augmentation/__init__.py

```diff
@@ -40,7 +40,7 @@
     RandomPerspective3D,
 )
 from .mix_augmentation import RandomMixUp, RandomCutMix
-from .container import VideoSequential
+from .container import VideoSequential, AugmentationSequential, Sequential
```

Comment:
I think we should start considering to add full path imports

Reply:
There was a fullpath import error with python==3.6. Full path import will result in some errors with the module initialisation orders.